nessie/externals/s3.py

```diff
@@ -204,14 +216,20 @@ def upload_from_url(url, s3_key, on_stream_opened=None):
         if on_stream_opened:
             on_stream_opened(response.headers)
         try:
-            s3_upload_args = {'ServerSideEncryption': 'AES256'}
+            s3_upload_args = {'ServerSideEncryption': app.config['LOCH_S3_ENCRYPTION']}
             if s3_url.endswith('.gz'):
                 s3_upload_args.update({
                     'ContentEncoding': 'gzip',
                     'ContentType': 'text/plain',
                 })
+            session = get_session()
             # smart_open needs to be told to ignore the .gz extension, or it will smartly attempt to double-compress it.
-            with smart_open.smart_open(s3_url, 'wb', ignore_extension=True, s3_upload_args=s3_upload_args) as s3_out:
+            with smart_open.open(
+                s3_url,
+                'wb',
+                ignore_ext=True,
```

Comment:
@sandeepmjay `ignore_ext` is right and `ignore_extension` is wrong?

Reply:
Smart open folks thought it would optimize to maximize the smartness with the change in 1.8.3

Reply:
Smart they are.